server/src/test/resources/templates/block_proof_template.json

```diff
@@ -0,0 +1,7 @@
+{
+  "block_item": {
+    "block_proof": {
+      "block": -1
+    }
+  }
+}
```

Comment:
EOF blank line

Reply:
Do you have a script or something that finds these missing EOLs?

Reply:
I just see them on the GitHub 
![image](https://github.com/user-attachments/assets/e57d127b-718f-4269-ad63-c06dbcb6d25a)

Reply:
Ah, ok

Reply:
Fixed